.circleci/config.yml

```diff
@@ -108,6 +108,44 @@ jobs:
             - ./.pytype
           key: v1-pytype
 
+  test:
+      docker:
+          - image: circleci/python:3.7.4-node
+            environment:
+              APP_CONFIG: config.TestingConfig
+              DATABASE_URL: postgresql://postgres@localhost/test
+              TEST_DATABASE_URL: postgresql://postgres@localhost/test
```

Comment:
You're still using wrong indentation and mixing and matching the amount of spaces. You can literally copy and paste the dredd job section and make edits. And please turn off auto indentation of your IDE.

Reply:
Use vim if the IDE is not stopping "intelligently" adding random indentation of spaces which it thinks is correct

Reply:
i fixed it

Reply:
IDE was pycharm was auto indenting on commit